nidata/multimodal/hcp/hcp.rst

```diff
@@ -1 +1,19 @@
-TODO: HCP needs a RST file.
\ No newline at end of file
+HCP 900 Subjects Release Data
+
+
+Notes
+-----
+The Human Connectome Project seeks to map out the region-to-region structural and functional connections in the human brain. The 900 Subjects Release Data includes imaging data from over 900 healthy adult subjects collected from 2012 to spring of 2015. The data set not only includes MR scans, but also over 95 subjects with resting-state and task MEG scans.
```

Comment:
Not sure what `region-to-region` means here... and I doubt that you do either :)  This should be in your own words, using technical words only if you understand them clearly.